djangocms_pageadmin/admin.py

```diff
@@ -75,22 +75,25 @@ def url(self, obj):
             url = obj.page.get_absolute_url(obj.language)
             formatted_url = format_html('<a href="{url}">{url}</a>', url=url)
```

Comment:
Just return `format_html('<a href="{url}">{url}</a>', url=url)`, this second format is unnecessary.